crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3386,6 +3386,24 @@ pub enum TypeParam {
     TypeVarTuple(TypeParamTypeVarTuple),
 }
 
+impl TypeParam {
+    pub fn name(&self) -> &Identifier {
```

Comment:
nit: this can be `const`

```suggestion
    pub const fn name(&self) -> &Identifier {
```